src/components/vistas/home/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './styles.module.scss';
+
+function Home() {
+  return (
+    <div className={styles.background}>
+      <div className={styles.home}>
+        <div className={styles.cont1}>
+          <div className={styles.containerHome}>
+            <h1 className={styles.title}>¡Hola!</h1>
+            <strong>Queremos darte una mano :)</strong>
+            <br />
+            <strong>¿Cómo podemos ayudarte hoy?</strong>
+          </div>
+        </div>
+        <hr className={styles.hr} />
+        <div className={styles.cont2}>
+          <div className={styles.containerButton}>
+            <button type="submit" className={styles.button1}>
+              QUIERO DONAR
+            </button>
+            <Link to="/medicine-selector">
+              {}
```

Comment:
Reemplaza esto por un comentario que diga para lo que se usará. Ejemplo: 
```suggestion
              {/* TODO: Aquí va el componente que ... */}
              {}
```